play-json/shared/src/main/scala/play/api/libs/json/Writes.scala

```diff
@@ -292,17 +329,22 @@ trait DefaultWrites extends LowPriorityWrites {
     def writes(d: java.util.Date): JsValue = JsString(new java.text.SimpleDateFormat(pattern).format(d))
   }
 
+  @deprecated("Use `defaultDateWrites`", "2.7.0")
+  object DefaultDateWrites extends Writes[Date] {
+    def writes(d: Date): JsValue = JsNumber(d.getTime)
+  }
+
   /**
    * Default Serializer java.util.Date -> JsNumber(d.getTime (nb of ms))
    */
-  implicit object DefaultDateWrites extends Writes[java.util.Date] {
-    def writes(d: java.util.Date): JsValue = JsNumber(d.getTime)
-  }
+  implicit def defaultDateWrites[T <: Date]: Writes[T] =
+    Writes[T] { d => JsNumber(d.getTime) }
 
   /**
    * Serializer for java.sql.Date
    * @param pattern the pattern used by SimpleDateFormat
    */
+  @deprecated("Use `dateWrites`", "2.7.0")
```

Comment:
Do you mean `defaultDateWrites`?

Reply:
[`dateWrites`](https://github.com/playframework/play-json/blob/6916289be1ea31828ab5886e602aaa9e295a1203/play-json/shared/src/main/scala/play/api/libs/json/Writes.scala#L328)